packages/angular/build/src/tools/esbuild/application-code-bundle.ts

```diff
@@ -48,8 +51,9 @@ export function createBrowserCodeBundleOptions(
     entryNames: outputNames.bundles,
     entryPoints,
     target,
-    supported: getFeatureSupport(target, isZonelessApp(polyfills)),
+    supported: getFeatureSupport(target, zoneless),
     plugins: [
+      createWasmPlugin({ allowAsync: zoneless, cache: sourceFileCache?.loadResultCache }),
```

Comment:
Shouldn't this be also done in the `createServerCodeBundle` it believe it will work for Node.js